packages/blockchain-wallet-v4-frontend/src/scenes/Register/template.tsx

```diff
@@ -260,17 +261,15 @@ const Register = ({
               </InfoItem>
             </CardInfo>
 
-            {showForm && (
+            {showForm || showWalletFormQuery ? (
```

Comment:
Can you do this here:

https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/2507#diff-b063896e9fb11559bbf2fe6c51fa9e3bR50

showForm={this.state.showForm || this.state.showWalletFormQuery}